python/setup.py

```diff
@@ -276,7 +276,6 @@ def run(self):
         classifiers=[
             'Development Status :: 5 - Production/Stable',
             'License :: OSI Approved :: Apache Software License',
-            'Programming Language :: Python :: 3.6',
```

Comment:
I guess we also need to update https://github.com/apache/spark/blob/33f19012dcdab1e4c880816b1377b2ad881bf1ae/python/setup.py#L275 ?